pkgs/data/fonts/line-awesome/default.nix

```
@@ -0,0 +1,32 @@
+{ lib, fetchzip }:
+
+let
+  version = "1.3.0";
+in
+fetchzip {
+  name = "line-awesome-${version}";
+
+  url = "https://maxst.icons8.com/vue-static/landings/line-awesome/line-awesome/${version}/line-awesome-${version}.zip";
+
+  hash = "sha256:0shxzjpxvicl5l39iqa89r7qry0kxwin3fkya5cgx484h6g300bi";
+
+  postFetch = ''
+    mkdir -p $out/share/fonts
+    unzip -j $downloadedFile ${version}/fonts/\*.ttf -d $out/share/fonts/truetype
+    unzip -j $downloadedFile ${version}/fonts/\*.woff -d $out/share/fonts/woff
+    unzip -j $downloadedFile ${version}/fonts/\*.woff2 -d $out/share/fonts/woff2
+  '';
+
+  meta = with lib; {
+    description = "Line Awesome - Replace Font Awesome with modern line icons";
```

```suggestion
    description = "Replace Font Awesome with modern line icons";
```

https://nixos.org/nixpkgs/manual/#sec-standard-meta-attributes